tools/ports/sdl2_mixer.py

```diff
@@ -16,14 +17,28 @@
   'sdl2_mixer-none-mt': {'SDL2_MIXER_FORMATS': [], 'PTHREADS': 1},
 }
 
+OPTIONS = {
+  'formats': 'A comma separated list of formats (ex: --use-port=sdl2_mixer:formats=ogg,mp3)'
+}
+
+SUPPORTED_FORMATS = {'ogg', 'mp3', 'mod', 'mid'}
+
+# user options (from --use-port)
+opts: Dict[str, Set] = {
+  'formats': set()
+}
+
 
 def needed(settings):
   return settings.USE_SDL_MIXER == 2
 
 
+def get_formats(settings):
+  return set(settings.SDL2_MIXER_FORMATS).union(opts['formats'])
```

Comment:
I think you can just write this as `opts['formats'].union(settings.SDL2_MIXER_FORMATS)` (i.e. `union` can take a list)

Reply:
Thanks. I changed it to `opts['formats'].union(settings.SDL2_MIXER_FORMATS)` (and update the similar code in sdl2_image as well).